### PR TITLE
[MUI-605]: Connect wallet for tablet

### PR DESCRIPTION
**Pull request by mr-teddy-bear**
# Description

Connect wallet for tablet

## Jira ticket(optional)

[MUI-605](https://minterestfinance.atlassian.net/jira/software/projects/MUI/boards/7?assignee=62591503eee0a9006ab98b74&selectedIssue=MUI-605)

## Type of change

Please pick options that are relevant.

- [ ] Bug fix OR refactoring (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature or refactoring that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by BerRossDolmat — APPROVED** [review]

**Review by l-polina — APPROVED** [review]

File: src/components/Navigation/Drawer.tsx [hunk 1]
```diff
@@ -40,11 +40,15 @@ const TitleBlock = styled('div')({
...
 
-const BottomButton = styled('div')({
+const BottomButton = styled('div')(({ theme }) => ({
   display: 'flex',
   alignItems: 'center',
   justifyContent: 'center',
-});
+
+  [theme.breakpoints.up('md')]: {
+    display: 'none',
```
Comment (gavrilin-remote) [inline src/components/Navigation/Drawer.tsx:49]: I don't like that way to hide button. This is highly unexpected.

Comment (gavrilin-remote) [reply src/components/Navigation/Drawer.tsx:49]: Better do not pass it from main-frontend and keep ui-kit component as much clear as we can

**Review by gavrilin-remote — CHANGES_REQUESTED** [review]

File: src/components/Navigation/Drawer.tsx (hunk 1 above)
Comment (BerRossDolmat) [reply src/components/Navigation/Drawer.tsx:49]: From one side, this components is not to be reused, probably. From other side, would be good to have this configurable from a place of use.
So probably some prop can be passed with failsafe default here?

File: src/components/Navigation/Drawer.tsx [hunk 2]
```diff
@@ -70,6 +79,7 @@ const DrawerMenu: FC<IDrawerProps> = ({
   isOpen,
   onClose,
   activeRoute,
+  hideExtraComponentOnTablet,
```
Comment (gavrilin-remote) [inline src/components/Navigation/Drawer.tsx:82]: I think will be better to pass props like showExtraComponent and control this from main-frontend

File: src/components/Navigation/Drawer.tsx [hunk 3]
```diff
@@ -108,7 +118,9 @@ const DrawerMenu: FC<IDrawerProps> = ({
             })}
           </List>
         </Box>
-        <BottomButton>{ExtraComponent && <ExtraComponent />}</BottomButton>
+        <BottomButton hideExtraComponentOnTablet={hideExtraComponentOnTablet}>
```
Comment (gavrilin-remote) [inline src/components/Navigation/Drawer.tsx:121]:
```suggestion
        {ExtraComponent && showExtraComponent && <BottomButton><ExtraComponent /></BottomButton>}
```

**Review by gavrilin-remote — CHANGES_REQUESTED** [review]

File: src/components/Navigation/Drawer.tsx (hunk 2 above)
Comment (mr-teddy-bear) [reply src/components/Navigation/Drawer.tsx]: Done

File: src/components/Navigation/Drawer.tsx (hunk 3 above)
Comment (mr-teddy-bear) [reply src/components/Navigation/Drawer.tsx]: Done

File: src/components/Navigation/Drawer.tsx (hunk 1 above)
Comment (mr-teddy-bear) [reply src/components/Navigation/Drawer.tsx]: Done

**Pull request merged**